codacy/requirements.lock

```diff
@@ -48,4 +48,4 @@ dependencies:
   repository: https://charts.codacy.com/stable
   version: 4.3.5
 digest: sha256:153c56c4e6398b2eecaa91f305189858583c09c9d23143386ecae78641dc1727
-generated: "2019-12-20T14:02:13.295509Z"
+generated: "2019-12-20T16:16:25.799537Z"
```

Comment:
Should this file be committed? 
Seems to be a generated file

Reply:
This file should be checked in because this is what helm will use to determine the versions of all the components to be installed. As part of a release, we do not perform a helm dep up command so we want to use the "frozen" versions, i.e. the versions in the .lock file.

Reply:
understood, thanks :D